### DISCUSSION TITLE
CSS Generator

### DISCUSSION THREAD

**Discussion by imhardikdesai** [scaleracademy/open-source-projects#91] [Ideas]
With this Project we can generate many type of css at one platform. For example Shape Generate, Gradient, color combination, Borders.

**Comment by itsmdasifraza** [comment]
Yaa, it will be very benifitial to frontend devs.

**Reply by imhardikdesai** [reply]
> Yaa, it will be very benifitial to frontend devs.

Yes brother you are right

**Comment by aaravdobariya** [comment]
We can also add custom js for each element to make it more productive.

**Comment by Dhruvbagadiya** [comment]
Great idea for Data Visualization , Shades and Color Gradiants

**Comment by AryanGoti18** [comment]
good idea for making a competitive projects

**Comment by URVESH1121** [comment]
It is helpful for Front-end developers. Good idea👌

**Comment by sshsrn** [comment]
Good idea, but there are already many platforms and services for it. 
With Figma, a designing tool, you can also generate the css from the design you make, but a dedicated portal just for getting css and for the animations like transitions and all will be cool!

**Reply by imhardikdesai** [reply]
Yes, this portal is best for generate static CSS.

**Comment by imdesaikeyur** [comment]
Yes, Good 💡 Idea. all CSS problem solutions in one place. also, those who have problems writing CSS can use this also.

**Comment by akashhapani** [comment]
Very Good Ideas for fast fronted development
